integrations/argocd/CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+# Port_Ocean 0.1.35 (2024-04-04)
+
+### Improvements
+
+- Reveresed the relation between cluster and namespace, and other general enhancements on blueprints (PORT-7550)
+- Updated the default mapping for deployment-history and kubernetes-resource to use the itemsToParse functionality and introduced a deprecation warning for these two kinds
+
```

Comment:
maybe there is a deprecation title that can be added, and add the deprecation announcement there